time/src/main/java/net/jqwik/time/DefaultDateArbitrary.java

```diff
@@ -70,6 +71,13 @@ public DateArbitrary monthLessOrEqual(Month max) {
 		return clone;
 	}
 
+	@Override
+	public DateArbitrary onlyMonths(Month... months) {
+		DefaultDateArbitrary clone = typedClone();
+		clone.allowedMonths = months;
+		return clone;
+	}
```

Comment:
How should I handle the value null?

Reply:
You mean if months is null?

That's not legal. But I don't think this is checked elsewhere.